### PR TITLE
Update GEOSwift.podspec

### PR DESCRIPTION
**Pull request by mojtabacazi**
Removed version restriction for `geos`

### CODE REVIEW

**Comment by vfn** [comment]
@mojtabacazi Can you please have a look at why Travis is failing? Cheers!

File: GEOSwift.podspec [hunk 1]
```diff
@@ -18,6 +18,6 @@ DESC
   s.source       = { :git => "https://github.com/GEOSwift/GEOSwift.git", :tag => s.version }
 
   s.source_files = "GEOSwift/*.{swift,h}"
-  s.dependency "geos", "3.5.0"
+  s.dependency "geos"
```
Comment (ablokker) [inline GEOSwift.podspec]: I would prefer you explicitly state the version here, in this case 3.5.1.  The "geos" 3.5.0 is using a svn repo, which is failing to checkout for me as well.

Comment (mojtabacazi) [reply GEOSwift.podspec]: then we have to set it to 3.6.2. I'm waiting for @vfn to give me access to `geos podspec` so I can update both. 
It's already updated here.
`source 'https://github.com/mojtabacazi/Specs.git'`

Comment (vfn) [reply GEOSwift.podspec]: @mojtabacazi please let me know what email you would like to use for accessing the podspecs

Comment (mojtabacazi) [reply GEOSwift.podspec]: @vfn same github account

Comment (vfn) [reply GEOSwift.podspec]: @mojtabacazi I can't see your email. It's probably set to private on your settings. https://github.com/settings/emails

**Comment by macdrevx** [comment]
I'm closing this since GEOSwift 3.0.0 is now available with support for GEOS 3.7.0. Let me know if you have any trouble with it.

**Pull request closed without merging**